query/autocomplete.js

```diff
@@ -59,23 +61,28 @@ query.filter( peliasQuery.view.boundary_rect );
 **/
 function generateQuery( clean ){
 
-  var vs = new peliasQuery.Vars( defaults );
+  const vs = new peliasQuery.Vars( defaults );
+
+  let logStr = '[query:autocomplete] [parser:addressit] ';
```

Comment:
oh, no, sorry, i interpreted my test code incorrectly.  Ignore me, apparently `+=` operates differently between `const` and `let`:

```
> const s = 'a';
undefined
> s += 'b';
'ab'
> s
'a'
> let t = 'a';
undefined
> t += 'b';
'ab'
> t
'ab'
```